content/en/docs/installation/kubernetes.md

```diff
@@ -141,6 +141,67 @@ The default cert-manager configuration is good for the majority of users, but a
 full list of the available options can be found in the [Helm chart
 README](https://artifacthub.io/packages/helm/cert-manager/cert-manager).
 
+## Installing with Operator Lifecycle Manager and OperatorHub.io
+
+Browse to the [cert-manager page on OperatorHub.io](https://operatorhub.io/operator/cert-manager),
+click the "Install" button and follow the installation instructions.
+
+Alternatively, [install OLM][] and [install the `kubectl operator` plugin][]
+from the [Krew Kubectl plugins index][] and then use that to install the cert-manager as follows:
+
+```sh
+operator-sdk olm install
+kubectl krew install operator
+kubectl operator install cert-manager -n operators --channel stable --approval Automatic
```

Comment:
Where can I find an explanation for the difference between `Automatic` and `Manual` approval?
Could this be added in the docs too?

Reply:
I added a link below.